driver_cpl/cime_config/config_component.xml

```diff
@@ -2974,14 +2974,6 @@
     <desc>Whether to clean the test after it is built/run</desc>
   </entry>
 
-  <entry id="CCSM_BASELINE">
```

Comment:
so will BASELINE_ROOT be added as a new xml variable in config_component.xml?
I am wondering if it was necessary to do this as part of this PR? could we have separated this into another PR?